hardware/Dummy.cpp

```diff
@@ -394,6 +396,9 @@ namespace http {
 			{
 				root["status"] = "OK";
 				root["title"] = "CreateVirtualSensor";
+				std::stringstream ss;
```

Comment:
Damn, that is probably my limit in C++ and my biggest concern as an embedded programmer. I expected the = operator not to perform extra action. Now that you point it out, I guess "returning" the local variable ID through the = operator won't fail after the function returns. Is that it ?